public/js/hipsterList.js

```diff
@@ -0,0 +1,114 @@
+
+function HipsterList() {
+//menu items
+ this.shopInfo =
+      {
+        "shopName": "The Coffee Connection",
+        "address": "123 Lakeside Way",
+        "phone": "16503600708",
+        "prices": 
```

Comment:
This is ok here but it would be better if you'd kept the json file and loaded it in using ajax